plugin-server/src/main/ingestion-queues/session-recording/services/replay-events-ingester.ts

```diff
@@ -138,6 +138,11 @@ export class ReplayEventsIngester {
                 if (replayRecord !== null) {
                     const asDate = DateTime.fromSQL(replayRecord.first_timestamp)
                     if (!asDate.isValid || Math.abs(asDate.diffNow('day').days) >= 7) {
+                        const eventTypes = rrwebEvents.map((event) => ({
+                            type: event.type,
+                            timestamp: event.timestamp,
```

Comment:
this will hopefully let me spot any patterns in the bad data, and figure out how I can safely capture a custom event instead of dropping here